docs/environment-variables.md

```diff
@@ -124,6 +109,24 @@ happens, subgraphs might process inconsistent data. Defaults to 250.
   this variable is set to any value, `graph-node` will still accept GraphQL
   subscriptions, but they won't receive any updates.
 
+### GraphQL caching
+
+- `GRAPH_CACHED_SUBGRAPH_IDS`: when set to `*`, cache all subgraphs (default behavior). Otherwise, a comma-separated list of subgraphs for which to cache queries.
+- `GRAPH_GRAPHQL_QUERY_TIMEOUT`: maximum execution time for a graphql query, in
+  seconds. Default is unlimited.
+   in the query cache. This should be kept small since the lookup time and the
+   cache memory usage are proportional to this value. Set to 0 to disable the cache.
+   Defaults to 1.
```

Comment:
Some copy pasting went wrong here, `GRAPH_GRAPHQL_QUERY_TIMEOUT` should be kept in the 'GraphQL' section.